algorithms/linfa-linear/src/ols.rs

```diff
@@ -267,7 +190,7 @@ mod tests {
     /// fit we only need to provide one point.
     #[test]
     fn without_intercept_fits_line_through_origin() {
-        let lin_reg = LinearRegression::new().with_intercept(false);
+        let lin_reg = LinearRegression::new().fit_intercept(false);
```

Comment:
can you keep the original name here, writing `LinearRegression::new().fit_intercept(false).fit(..)` sounds weird

Reply:
Should I change it in logistic regression as well?

Reply:
yes to keep the naming consistent

Reply:
Turns out the name in logistic was already `with_intercept`